hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -132,17 +163,100 @@ private HederaAccountCustomizer asCustomizer(CryptoCreateTransactionBody op) {
+
+                final JKey jKeyFromAlias = asFcKeyUnchecked(keyFromAlias);
+                customizer
+                        .key(jKeyFromAlias)
+                        .memo(AUTO_MEMO)
+                        .autoRenewPeriod(THREE_MONTHS_IN_SECONDS)
+                        .expiry(txnCtx.consensusTime().getEpochSecond() + THREE_MONTHS_IN_SECONDS)
+                        .isReceiverSigRequired(false)
+                        .isSmartContract(false)
+                        .alias(op.getAlias());
+            }
+        } else if (!op.getAlias().isEmpty() && op.hasKey()) {
+            if (!aliasManager.lookupIdBy(op.getAlias()).equals(MISSING_NUM)) {
+                throw new InvalidTransactionException(INVALID_ALIAS_KEY);
+            }
+            if (!op.getKey().getECDSASecp256K1().isEmpty()) {
+                final var recoveredEvmAddressFromPrimitiveKey =
+                        recoverAddressFromPubKey(op.getKey().getECDSASecp256K1().toByteArray());
+
+                if (recoveredEvmAddressFromPrimitiveKey != null
+                        && !aliasManager
+                                .lookupIdBy(
+                                        ByteString.copyFrom(recoveredEvmAddressFromPrimitiveKey))
+                                .equals(MISSING_NUM)) {
+                    throw new InvalidTransactionException(INVALID_ALIAS_KEY);
+                }
+            }
+            customizer
+                    .key(asFcKeyUnchecked(op.getKey()))
+                    .memo(AUTO_MEMO)
+                    .autoRenewPeriod(THREE_MONTHS_IN_SECONDS)
+                    .expiry(txnCtx.consensusTime().getEpochSecond() + THREE_MONTHS_IN_SECONDS)
+                    .isReceiverSigRequired(false)
+                    .isSmartContract(false)
+                    .alias(op.getAlias());
+        } else {
+            /* Note that {@code this.validate(TransactionBody)} will have rejected any txn with an invalid key. */
+            if (!op.getKey().getECDSASecp256K1().isEmpty()) {
+                if (!aliasManager.lookupIdBy(op.getKey().getECDSASecp256K1()).equals(MISSING_NUM)) {
+                    throw new InvalidTransactionException(INVALID_ALIAS_KEY);
+                }
+
+                final var recoveredEvmAddressFromPrimitiveKey =
+                        recoverAddressFromPubKey(op.getKey().getECDSASecp256K1().toByteArray());
+
+                if (recoveredEvmAddressFromPrimitiveKey != null
+                        && !aliasManager
+                                .lookupIdBy(
+                                        ByteString.copyFrom(recoveredEvmAddressFromPrimitiveKey))
+                                .equals(MISSING_NUM)) {
+                    throw new InvalidTransactionException(INVALID_ALIAS_KEY);
+                }
+                customizer.alias(ByteString.copyFrom(recoveredEvmAddressFromPrimitiveKey));
+            }
+
+            final JKey key = asFcKeyUnchecked(op.getKey());
+            customizer
```

Comment:
There's differences and similarities between this customizer set and the other  cases in terms of fields and values.
We should pull out the shared set to be at the beginning and let the if/else set or override the special a cases.

e.g.
- `isDeclinedReward` and `maxAutomaticAssociations ` are only set here
- this is the only case that sets expiry to `expiry`. The other cases set it as `txnCtx.consensusTime().getEpochSecond() + THREE_MONTHS_IN_SECONDS`